ideaforce/src/main/java/com/emc/ideaforce/utils/Utils.java

```diff
@@ -3,7 +3,7 @@
 public class Utils {
     public static final String PWD_PRIVELEGE = "CHANGE_PASSWORD_PRIVILEGE";
     public static final String REG_USER_ROLE = "ROLE_USER";
-    public static final String ADMIN_ROLE = "ADMIN";
+    public static final String ADMIN_ROLE = "ROLE_ADMIN";
```

Comment:
Is default admin user also having this role?

Reply:
yes